### PR TITLE
CAW21-35 Enable and configure book module for subsites

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Install and configure book module to be the subsite structure

# Need Review By (Date)
- soon

# Urgency
- medium

# Steps to Test
1. grab the latest `ace-gryphon` 2.x branch (if you need help setting that up, let me know)
2. `composer require su-sws/caw_profile:dev-CAW21-35 --no-update`
3. `composer update -n`
4. `blt drupal:sync --site=cardinalatwork`
5. log into your local site
6. create some basic pages and configure them as subsites.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Comment by imonroe** [comment]
So, I got acegryphon installed locally, and I got it working and could log in.  I got to the step where you say to:
`blt drupal:sync --site=cardinalatwork`

That seems to run ok, but then fails at the end with this error: 

image

**Comment by pookmish** [comment]
Hmm. I'll try to work through that. I might just have to enable the book module on the prod site to prevent that error.

**Comment by pookmish** [comment]
@imonroe try again. you might have to drop your database first though. `drush @cardinalatwork.local sql-drop -y; blt drupal:sync --site=cardinalatwork`

File: themes/caw_theme/package.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
+{
+	"name": "stanford_module_example",
```
Comment (imonroe) [inline themes/caw_theme/package.json:2]: should probably change this.

File: themes/caw_theme/package.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+	"version": "0.1.0",
+	"repository": "https://github.com/SU-SWS/stanford_module_example",
```
Comment (imonroe) [inline themes/caw_theme/package.json:4]: also this.

File: themes/caw_theme/package.json [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+		"uglifyjs-webpack-plugin": "^2.1.2",
+		"url-loader": "^1.1.2",
+		"webpack": "^4.29.6",
+		"webpack-assets-manifest": "^3.1.1",
+		"webpack-cli": "^3.3.0",
+		"webpack-dev-server": "^3.2.1",
+		"webpack-fix-style-only-entries": "^0.2.1",
+		"webpack-manifest-plugin": "^2.0.4"
+	},
+	"description": "=========================== How to Use Grunt with Stanford Module Example ===========================",
```
Comment (imonroe) [inline themes/caw_theme/package.json:52]: likely this as well.

**Review by imonroe — APPROVED** [review]
This installs correctly, functions as described.  Looks pretty nice too.  Tests are included, and the tests pass.  There are a couple of trivial things where some default text should be replaced as noted, but other than that, it's good to go.  I'm going to approve, but not merge, since this is a big PR.  Please correct the couple of default strings and merge at your discretion.

**Pull request merged**